### ISSUE TITLE
Unicycler hybrid assembly not allocated intended resources

### ISSUE THREAD

**Issue by marcmtk** [bactopia/bactopia#320]
Thank you for the massive undertaking of this package and all the contributed models to nf-core. I've learned a lot from poking around your source code :smile: 

I'm trying hybrid assembly with bactopia for the first time.

My run restarted the assemble_genome process after 1h, but as far as i can tell from the conf/base.config file it is supposed to be allocated two hours and 32GB memory. https://github.com/bactopia/bactopia/blob/b022505e8a9d0a22a92c3b35e7269012343b01d6/conf/base.config#L42-L43

My command
`bactopia --R1 R1.fastq.gz --R2 R2.fastq.gz --SE long.fastq.gz --hybrid --sample Klebsiella --run_name Klebsiella --genome_size 5300000 --datasets ~/bactopia/datasets/ --species "Klebsiella pneumoniae" --run_checkm --full_tree --use_bakta --bakta_db ~/databases/bakta/db/`

The log says

> May-18 14:44:11.411 [Task monitor] DEBUG n.processor.TaskPollingMonitor - Task completed > TaskHandler[id: 4; name: BACTOPIA:ASSEMBLE_GENOME (Klebsiella); status: COMPLETED; exit: -; error: nextflow.exception.ProcessException: Process exceeded running time limit (1h); workDir: ~/Desktop/bacterial_analysis/20220419/bactopia/work/32/e9a30e1715495c8656d5716b066177]

I believe the issue may be that sample_type isn't defined and i wasn't adding `--use_unicycler`. I'm not sure which variable should be subbed in instead of sample_type, if any.

**Comment by rpetit3** [comment]
Hi @marcmtk 

Nice catch! Looks like I failed to make this change between V1 and V2. I think a have a fix in place for you (https://github.com/bactopia/bactopia/commit/37e63e8e58386fba593f4b4dd888b30815cc7952)

It looks like you might already be using my dev build, if not here's details
```
mamba create -n bactopia-dev -c rpetit3 -c conda-forge -c bioconda bactopia
conda activate bactopia-dev
```

Let me know if you spot anything else! Also, let me know if you have any feedback or things you'd like to see included.

Cheers,
Robert

**Comment by rpetit3** [comment]
Hi @marcmtk 

I released v2.1.0 (https://github.com/bactopia/bactopia/releases/tag/v2.1.0) which should fix includes the changes from the dev build you were using. If your problem persists, please feel free to reopen this issue!

If you have some time to spare, please consider filling out my Bactopia survey.  I use the feedback to direct the future developments of Bactopia. (_If you've already filled it out, thank you so much!_)

https://forms.gle/PTgUxEBwCsCdbQjr7

Cheers,
Robert
